adcc/backends/test_backends_polembed.py

```diff
@@ -82,3 +88,53 @@ def template_pe_formaldehyde(self, basis, method, backend):
             state.pe_ptlr_correction,
             atol=1e-5
         )
+
+    def template_pe_coupling_formaldehyde(self, basis, method, backend):
+        if method != "adc2":
+            pytest.skip("")
+        basename = f"formaldehyde_{basis}_pe_{method}"
+        tm_result = tmole_data[basename]
+        pe_options = {"potfile": pe_potentials["fa_6w"]}
+        scfres = cached_backend_hf(backend, "formaldehyde", basis,
+                                   pe_options=pe_options)
+        assert_allclose(scfres.energy_scf, tm_result["energy_scf"], atol=1e-8)
+
+        matrix = adcc.AdcMatrix(method, scfres)
+        solvent = AdcExtraTerm(matrix, {'ph_ph': block_ph_ph_0_pe})
+
+        with pytest.raises(NotImplementedError):
```

Comment:
If you do operators properly this should raise a TypeError automatically.

Reply:
That's right 😄 Done.

Reply:
Damn I hate Python... now having the "correct" implementation, i.e., returning `NotImplemented`, Python tries to be super clever and inverts the argument order automatically to `matrix += solvent` 🤯
I don't know how one can disable this 👎